### PR TITLE
Intermediate imgs2

### PR DESCRIPTION
**Pull request by Roccy**
Added intermediate results:
* Cropped with skin detection
* Skeleton
* Skeleton with keypoints

### CODE REVIEW

**Comment by jorenham** [comment]
Would be nice if the buttons are disabled when processing
Also some indication of at what step of the pipeline you are at the moment, with e.g a progress bar with some text

**Comment by jorenham** [comment]
Maybe you could use this
https://github.com/stepstone-tech/android-material-stepper

**Comment by Roccy** [comment]
I'll check it out, thanks. @jorenham

File: FingerBlox/app/src/main/java/org/fingerblox/fingerblox/ImageDisplayActivity.java [hunk 1]
```diff
@@ -324,10 +325,15 @@ public static Mat jsonToMat(String json){
         return mat;
     }
 
+    public void onNewIntent(Intent intent) {
+        ImageView mImageView = (ImageView) findViewById(R.id.image_view);
```
Comment (jorenham) [inline FingerBlox/app/src/main/java/org/fingerblox/fingerblox/ImageDisplayActivity.java:329]: might be better to make this a field

**Comment by Roccy** [comment]
Closing this one, started rewite using the new UI.

File: FingerBlox/app/src/main/java/org/fingerblox/fingerblox/ImageDisplayActivity.java (hunk 1 above)
Comment (Roccy) [reply FingerBlox/app/src/main/java/org/fingerblox/fingerblox/ImageDisplayActivity.java:329]: Wut? It is missing an @Override tag though

**Pull request closed without merging**